policies/code-of-conduct.md

```diff
@@ -30,9 +30,12 @@ Welcome to the Code of Conduct (CoC) for the Commonhaus Foundation (CF). We're d
 
 ## Our Code of Conduct
 
-The CF adheres to the [Contributor Covenant v1.4](#contributor-covenant-code-of-conduct) as its foundational Code of Conduct (CoC), embracing a commitment to create a welcoming and inclusive environment. This CoC underpins all foundation activities and is a requirement for new projects within three months of joining.
+The CF is dedicated to creating a welcoming and inclusive environment. Our Code of Conduct (CoC), derived from the [Contributor Covenant v1.4](#contributor-covenant-code-of-conduct), sets the standard for behavior within our community.
 
-Leadership across the CF, including [Council members (CFC)][cfc] and project leaders, are [expected](#leadership-expectations) to respect and uphold this CoC to foster an environment of respect and collaboration. The [Code of Conduct Panel (CoCP)](#code-of-conduct-panel) provides an avenue for appeals or escalations, if necessary.
+Leadership across the CF, including [Councilors][cfc] and project representatives, are [expected](#leadership-expectations) to respect and uphold this CoC to foster an environment of respect and collaboration.
+The [Code of Conduct Panel (CoCP)](#code-of-conduct-panel) provides an avenue for appeals or escalations, if necessary.
+
+New projects are expected to adopt this CoC within three months of joining.
```

Comment:
Does this mean projects with their own CoC need to adopt the CF CoC?

Reply:
They would have to at least accept the escalation reporting structure, yes. If (at the project level) they want to define their own processes, they can.